migration-roadmap/status-light.md

```diff
@@ -0,0 +1,259 @@
+# Status Light migration roadmap
+
+## Component specifications
+
+### CSS
+
+<details>
+<summary>CSS selectors</summary>
+
+- `.spectrum-StatusLight`
+- `.spectrum-StatusLight--accent:before`
+- `.spectrum-StatusLight--blue:before`
+- `.spectrum-StatusLight--celery:before`
```

Comment:
Brown, cinnamon, pink, turquoise, and silver were all the new selectors in S2. This looks to be the selectors probably from S1 instead.